src/components/addProjectDialog.tsx

```diff
@@ -94,196 +95,198 @@ const AddProjectDialog: FunctionComponent<AddProjectDialogProps> = ({ open, hand
       maxWidth={'md'}
     >
       <DialogTitle>Projekt erstellen</DialogTitle>
```

Comment:
Actually I only added Lines 98 and 289, don't know why git is displaying all those as changes.